File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int n=s.size();
        int cnt=0; 
        int i=n-1;
        
        while(i>=0){
            if(s[i]==' ') i--;
            else break; 
        }
        
        for(int k=i; k>=0; k--){
            if(s[k]!=' ') cnt++;
            else break;
        }
        
        return cnt;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.go
func lengthOfLastWord(s string) int {
    
    n := len(s)
    i := n-1 
    var cnt int
    
    for i>=0 && s[i]==' '{
        i--
    }
    
    for i>=0 {
        cnt++
    
        if s[i]==' '{
            cnt--
            break
        }
        
        i--
    }
    
    return cnt
    
}